cmis_web_proxy_alf/static/src/js/form_widgets.js

```diff
@@ -22,9 +22,6 @@ form_widgets.CmisObjectWrapper.include({
 
     get_preview_url : function(){
       var _url =  this._super.apply(this, arguments);
-      if (_url) {
-          return _url;
-      }
```

Comment:
don't we break the inheritance here in a environment with multiple backend that are not linked to alfresco?